samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.Ignore();
```

Comment:
> Why wouldn't the sample still work via the c.Skip() method that lives in the event?

@HaoK there's no such thing. If you mean `SkipToNextMiddleware()`/`SkipHandler()`, it's no longer used in the JWT bearer middleware because it doesn't implement `IAuthenticationRequestHandler` and so can't have any influence on the request processing. I replaced both `SkipToNextMiddleware()`/`SkipHandler()` by `SkipAuthentication()` in my PR.